src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/accelerators/CropGrowthAccelerator.java

```diff
@@ -58,7 +58,7 @@ private boolean grow(Block machine, BlockMenu inv, Block crop) {
 
         if (ageable.getAge() < ageable.getMaximumAge()) {
             for (int slot : getInputSlots()) {
-                if (SlimefunUtils.isItemSimilar(inv.getItemInSlot(slot), organicFertilizer, false, false)) {
+                if (SlimefunUtils.compareItem(inv.getItemInSlot(slot), organicFertilizer)) {
```

Comment:
This compares with an unobtainable blank organic fertilizer item.
So this would break this machine.